test/controllers/ingredients_controller_test.rb

```diff
@@ -1,50 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Why are we deleting the test? :(

Reply:
This test file was auto-generated by the rails g scaffold command. Strange enough, it's not working since it says the ingredients resource is nil. The short version is: I deleted it so I could create the PR. :(